install.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+# Check if a package is installed
+package_installed() {
+    package="$1"
+    command -v "$package" >/dev/null 2>&1
```

Comment:
One more thing, I am wondering what `command` is? I am not saying it is wrong. I haven't seen this kind of use before that is why I am asking. Could you please explain!

Reply:
The command -v command takes the name of a command as an argument (e.g., vim, tmux, git) and check if it exsists and if it accesable